notebook.ipynb

```diff
@@ -155,6 +155,8 @@
     "import numpy as np\n",
     "\n",
     "\n",
+    "np.random.seed(0)\n",
```

Comment:
You are setting the seed in line 158, but in line 144 we have already model output?

Reply:
It seems that the cell output is stored before the cell content. As you can see here, we set the random seed before we create the random signal and call the model:

![image](https://user-images.githubusercontent.com/10383417/155493793-284e5ca9-9a7f-497f-a323-daf30d14d766.png)

Reply:
Ah, yes, you are right, that is stored before.

As I said before, reviewing Jupyter notebook code is such fun ;)